docs/websockets.rst

```diff
@@ -125,22 +125,28 @@ The messages sent by rotki when a user is logging in and a db upgrade is happeni
 - ``target_version``: The target migration version. When this will have been reached and finished, the migrations will end.
 
 
-EVM Addresses Migrations
-============================
+EVM Accounts Detection
+=======================
+
+If there are new evm accounts detected (due to a data migration, user-initiated detection, periodic detection, etc.) then the frontend will be notified of any new EVM accounts that were detected. This is done via the following message.
 
-At the user DB migrations when a new evm chain is introduced rotki will do a migration that will add any addresses used in mainnet to the new evm chain. At the same time it needs to notify the frontend that new evm chain/ evm address combination was added at migration so the frontend can do extra actions such as detecting tokens. The message is simple and just contains the list of migrated addresses if any.
+- ``"address"``: Address of a account.
+- ``"evm_chain"``: Chain of a account.
 
 ::
 
     {
-        "type": "evm_address_migration",
-        "data": [{
-            "evm_chain": "optimism",
-            "address": "0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12"
-        }, {
-            "evm_chain": "optimism",
-            "address": "0xFeebabE6b0418eC13b30aAdF129F5DcDd4f70CeA"
-        }]
+        "type": "new_evm_accounts_detected",
```

Comment:
rename: evm_accounts_detection

Keeping the names consistent with the previous one.

Reply:
you have not renamed this here yet